src/extra/widgets/chart/lv_chart.c

```diff
@@ -1202,7 +1202,8 @@ static void draw_series_bar(lv_obj_t * obj, lv_draw_ctx_t * draw_ctx)
     lv_draw_rect_dsc_t col_dsc;
     lv_draw_rect_dsc_init(&col_dsc);
     lv_obj_init_draw_rect_dsc(obj, LV_PART_ITEMS, &col_dsc);
-    col_dsc.bg_grad_dir = LV_GRAD_DIR_NONE;
+    col_dsc.bg_grad.dir = LV_GRAD_DIR_NONE;
+    col_dsc.bg_grad.dither = LV_DITHER_NONE;
```

Comment:
Is this line required if `bg_grad.dir = LV_GRAD_DIR_NONE;`?

Reply:
No, it's a copy & paste error. Removed.